versions.lock

```diff
@@ -170,24 +188,33 @@ org.apache.spark:spark-sketch_2.11:2.4.4 (2 constraints: 981bd4f5)
 org.apache.spark:spark-sql_2.11:2.4.4 (1 constraints: 1e0d0037)
 org.apache.spark:spark-tags_2.11:2.4.4 (8 constraints: 036fa69d)
 org.apache.spark:spark-unsafe_2.11:2.4.4 (2 constraints: f11bc213)
-org.apache.thrift:libfb303:0.9.3 (3 constraints: 6725fac0)
-org.apache.thrift:libthrift:0.9.3 (5 constraints: 71415452)
+org.apache.thrift:libfb303:0.9.3 (3 constraints: 27289d07)
+org.apache.thrift:libthrift:0.9.3 (7 constraints: c75ac7de)
+org.apache.twill:twill-api:0.6.0-incubating (2 constraints: e422b4e1)
+org.apache.twill:twill-common:0.6.0-incubating (4 constraints: 3d467991)
+org.apache.twill:twill-core:0.6.0-incubating (1 constraints: d70f9d7c)
+org.apache.twill:twill-discovery-api:0.6.0-incubating (3 constraints: 25345d4c)
+org.apache.twill:twill-discovery-core:0.6.0-incubating (2 constraints: 332039f9)
+org.apache.twill:twill-zookeeper:0.6.0-incubating (3 constraints: 94341288)
 org.apache.xbean:xbean-asm6-shaded:4.8 (2 constraints: 2419a30f)
 org.apache.yetus:audience-annotations:0.11.0 (1 constraints: c40eb364)
-org.apache.zookeeper:zookeeper:3.4.6 (11 constraints: 18a71f48)
+org.apache.zookeeper:zookeeper:3.4.6 (12 constraints: 54b50652)
 org.checkerframework:checker-qual:2.8.1 (2 constraints: 1a1a3944)
 org.codehaus.jackson:jackson-core-asl:1.9.13 (11 constraints: 8091cd06)
 org.codehaus.jackson:jackson-jaxrs:1.9.13 (2 constraints: 821bca9d)
 org.codehaus.jackson:jackson-mapper-asl:1.9.13 (11 constraints: e18d325f)
 org.codehaus.jackson:jackson-xc:1.9.13 (2 constraints: 821bca9d)
-org.codehaus.janino:commons-compiler:3.0.9 (3 constraints: 0a2837cc)
-org.codehaus.janino:janino:3.0.9 (2 constraints: 3f1c6304)
+org.codehaus.janino:commons-compiler:3.0.9 (2 constraints: a41a546f)
+org.codehaus.janino:janino:3.0.9 (1 constraints: d90e817c)
 org.codehaus.jettison:jettison:1.1 (4 constraints: a84e24a9)
 org.codehaus.mojo:animal-sniffer-annotations:1.17 (1 constraints: ed09d8aa)
-org.datanucleus:datanucleus-api-jdo:3.2.6 (2 constraints: 5a1d241c)
-org.datanucleus:datanucleus-core:3.2.10 (5 constraints: 4a4401e0)
-org.datanucleus:datanucleus-rdbms:3.2.9 (2 constraints: 601db41c)
+org.datanucleus:datanucleus-api-jdo:4.2.4 (2 constraints: 591df91b)
+org.datanucleus:datanucleus-core:4.1.17 (5 constraints: 584455e8)
+org.datanucleus:datanucleus-rdbms:4.1.19 (2 constraints: 911dec32)
+org.datanucleus:javax.jdo:3.2.0-m3 (1 constraints: 030ea249)
 org.eclipse.jdt:core:3.1.1 (1 constraints: b40a38d8)
+org.eclipse.jetty.aggregate:jetty-all:7.6.0.v20120127 (1 constraints: 260ea577)
+org.eclipse.jetty.orbit:javax.servlet:3.0.0.v201112011016 (1 constraints: dd0e53b1)
```

Comment:
Are these servlet classes needed?

Reply:
Seems not necessary, they're introduced by hive-common, I will remove them.